lib/libsys/intro.2

```diff
@@ -220,34 +222,31 @@ on the file system.
 .It Er 29 ESPIPE Em "Illegal seek" .
 An
 .Xr lseek 2
-system call was issued on a socket, pipe or
-.Tn FIFO .
+system call was issued on a socket, pipe or FIFO.
 .It Er 30 EROFS Em "Read-only file system" .
 An attempt was made to modify a file or directory
 on a file system that was read-only at the time.
 .It Er 31 EMLINK Em "Too many links" .
-Maximum allowable hard links to a single file has been exceeded (limit
-of 32767 hard links per file).
+Maximum allowable hard links to a single file has been exceeded
+.Pq limit of 32767 hard links per file .
```

Comment:
I'd drop the number entirely in a separate commit. The limit is UFS_LINK_MAX (65500) on UFS, UINT32_MAX on fuse, and INT_MAX on tmpfs.

Reply:
I think that's really cool/helpful information. How's this?
```
.Po
.Va UFS_LINK_MAX on
.Xr ufs 4 ,
.Va UINT32_MAX on
.Xr fusefs 4 ,
and
.Va INT_MAX on
.Xr tmpfs 4
.Pc .
```
If ok, would you prefer that as a separate commit or included into the cleanups?

Reply:
> I think that's really cool/helpful information. How's this?
> 
> ```
> .Po
> .Va UFS_LINK_MAX on
> .Xr ufs 4 ,
> .Va UINT32_MAX on
> .Xr fusefs 4 ,
> and
> .Va INT_MAX on
> .Xr tmpfs 4
> .Pc .
> ```
> 
> If ok, would you prefer that as a separate commit or included into the cleanups?

I think let's do ot as a new PR when I've landed this one...